server/plugin/client_test.go

```diff
@@ -339,11 +318,8 @@ func TestCall(t *testing.T) {
 
 func TestGetGitRepositories(t *testing.T) {
 	defer monkey.UnpatchAll()
-	p := Plugin{}
 	mockAPI := &plugintest.API{}
-	p.API = mockAPI
-	c := InitClient(&p)
-	p.Client = c
+	p := setupTestPlugin(mockAPI)
```

Comment:
```suggestion
	p := setupTestPlugin(&plugintest.API{})
```
There's no need to create an extra variable. Change this everywhere wherever possible.